.cursor/rules/calypso-client-rules.mdc

```diff
@@ -58,24 +58,33 @@ Before responding to any request, follow these steps:
 
 - Don't use `&--` & `&__` selectors and write full name when defining styles.
 - Try to always use RTL specific styles. For example, instead of margin-left, use margin-inline-start.
+- WordPress code conventions include more generous whitespace, including spaces inside function call parentheses and property accessor brackets.
```

Comment:
That works, but perhaps @jsnajdr might have more tips for describing the WP whitespace specificities better?